electricitymap/contrib/lib/models/event_lists.py

```diff
@@ -87,14 +160,11 @@ def merge_production_breakdowns(
         Sources will be aggregated in a comma-separated string. Ex: "entsoe, eia".
         There should be only one zone in the list of production breakdowns.
         """
-        if len(ungrouped_production_breakdowns) == 0:
-            return ProductionBreakdownList(logger)
-        if all(
-            len(breakdowns.events) == 0
-            for breakdowns in ungrouped_production_breakdowns
+        production_breakdowns = ProductionBreakdownList(logger)
+        if ProductionBreakdownList.is_completly_empty(
+            ungrouped_production_breakdowns, logger
```

Comment:
Pylance is throwing the following error here:
```Argument of type "List[ProductionBreakdownList]" cannot be assigned to parameter "ungrouped_events" of type "List[MergeableList]" in function "is_completly_empty"
  "List[ProductionBreakdownList]" is incompatible with "List[MergeableList]"
    TypeVar "_T@list" is invariant
      "ProductionBreakdownList" is incompatible with "MergeableList"PylancereportGeneralTypeIssues
```
So I think we need to accept the input types for the is_completly_empty function to accept the ProductionBreakdownList and ExchangeList instead.

Reply:
I have an issue with that approach as it means that the parent class is aware of what the child is doing which is not a good idea. I will see if I can fix it somehow.

Reply:
Apparently using Sequence is the best fix.
https://stackoverflow.com/questions/69891672/python-typing-how-to-handle-list-type-hinting-with-different-element-type-in-su